framework/libra-framework/sources/modified_source/stake.move

```diff
@@ -188,6 +188,31 @@ module diem_framework::stake {
         fees_table: Table<address, Coin<LibraCoin>>,
     }
 
+    #[view]
+    /// @return: tuple
+    /// - u64: number of proposals
+    /// - address: the validator
+    public fun get_highest_net_proposer(): (u64, address) acquires ValidatorSet,
```

Comment:
Could it be that we are going to miss to include some validators in case some of them have equal net value ?